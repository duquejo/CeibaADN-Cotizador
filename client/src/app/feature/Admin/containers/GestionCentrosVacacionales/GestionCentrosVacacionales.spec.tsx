import * as React from 'react';
import { ShallowWrapper, shallow } from 'enzyme';
import { GestionCentrosVacacionales } from './index';
import { SinonStub } from 'sinon';

describe('GestionCentrosVacacionales Test', () => {

  let componentWrapper: ShallowWrapper;
  let componentProps: React.ComponentProps<typeof GestionCentrosVacacionales> & {
    listarCentrosVacacionales: SinonStub;
    agregarNuevoCentroVacacional: SinonStub;
    actualizarCentroVacacional: SinonStub;
    eliminarCentroVacacional: SinonStub;    
  };

  it('DeberÃ­a renderizar el componente', () => {
    componentWrapper = shallow( <GestionCentrosVacacionales { ...componentProps } /> );
    expect(componentWrapper).toMatchSnapshot();
  });

});
