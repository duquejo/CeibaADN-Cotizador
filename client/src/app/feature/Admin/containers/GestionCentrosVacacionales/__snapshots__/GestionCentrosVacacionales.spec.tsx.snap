// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GestionCentrosVacacionales Test Debería renderizar el componente 1`] = `
<Fragment>
  <h2>
    Gestión de centros vacacionales 
    <span
      aria-label="Centros vacacionales"
      role="img"
      title="Centros vacacionales"
    >
      ⛱️
    </span>
  </h2>
  <p>
    Administre los centros vacacionales que los usuarios tendrán a su disposición en el cotizador.
  </p>
  <styled.div>
    <styled.div>
      <CrearCentroVacacional />
    </styled.div>
    <styled.div>
      <AccionesCentrosVacacionales />
    </styled.div>
  </styled.div>
  <styled.hr />
</Fragment>
`;
